pkgs/flox/default.nix

```diff
@@ -1,11 +1 @@
-{
-  self,
-  capacitated,
-  ...
-}:
-{
-  type = "derivation";
-  meta = capacitated.flox.packages.flox.meta;
-  inherit (capacitated.flox.packages.flox) outputs out man outPath outputName drvPath name system;
-  fromCatalog = self.evalCatalog.stable.flox;
-}
```

Comment:
@tomberek I cant quite tell what was the reason behind this

Reply:
The point was to remove the need to download flox source (via git) in order to install flox. It makes it a "lazy derivation": https://github.com/NixOS/nixpkgs/blob/master/lib/derivations.nix#L53

So this PR just broke some of our installation methods.